conda_forge_tick/auto_tick.py

```diff
@@ -192,13 +192,7 @@ def run(
 
     # TODO: run this in parallel
     feedstock_dir, repo = get_repo(
-        fctx=feedstock_ctx,
-        branch=branch_name,
-        feedstock=feedstock_ctx.feedstock_name,
-        protocol=protocol,
-        pull_request=pull_request,
-        fork=fork,
-        base_branch=base_branch,
+        fctx=feedstock_ctx, branch=branch_name, base_branch=base_branch
```

Comment:
I get why the API was changed here, given that we never used non-default values and will never do so either. We should at some point clean up the keywords on the surrounding function as well, but not today!